lib/Driver/Driver.cpp

```diff
@@ -695,6 +697,11 @@ Driver::buildCompilation(const ToolChain &TC,
 
   buildJobs(TopLevelActions, OI, OFM.get(), TC, *C);
 
+  if (DriverPrintDerivedOutputFileMap) {
+    C->getDerivedOutputFileMap().dump(llvm::errs(), true);
```

Comment:
Good call on this dump flag. I suggest printing to `llvm::outs()` for slightly easier testing.

Reply:
Done.